Feature/merge test context (#1085)
bUnit-dev/bUnit: MIGRATION.md

The `Fake` prefix has been replaced with `Bunit` in many test doubles. For example, `FakeNavigationManager` is now `BunitNavigationManager`. If you reference any of these types explicitly, you need to update your code.

### Renamed `AddTestAuthorization` to `AddAuthorization`
The `AddTestAuthorization` method on `TestContext` has been renamed to `AddAuthorization`. If you used `AddTestAuthorization`, you should replace it with `AddAuthorization`.

## Merged `TestContext` and `TestContextBase`

> The `TestContext` and `TestContextBase` classes have been merged into a single `TestContext` class. If you used either of these classes, you should replace them with `TestContext`.

Reviewer: ```suggestion
The `TestContext` and `TestContextBase` classes have been merged into a single `TestContext` class. All references to `TestContextBase` should replace them with `TestContext` to migrate.
```
